en-devonfw.dict

```diff
@@ -255,9 +253,9 @@ Modularization
 Moq
 MrChecker
 MrChecker’s
```

Comment:
please change to simple single quote, it seems some strange quote here

Reply:
fixed it